src/modules/reading-room/services/reading-room/reading-room.service.const.ts

```diff
@@ -1 +1 @@
-export const READING_ROOM_SERVICE_BASE_URL = '/api/proxy/spaces';
+export const READING_ROOM_SERVICE_BASE_URL = 'api/proxy/spaces';
```

Comment:
Die wordt meegegeven aan `ky.create({ prefixUrl: ... })` of werkt dat ook zonder de leading slash?

Reply:
it also works without, but it looks better with the leading slash
i had some trouble getting it to work because i was passing the path twice.